make/common/templates/clair/config.yaml

```diff
@@ -16,8 +16,10 @@ clair:
     # Deadline before an API request will respond with a 503
     timeout: 300s
   updater:
-    interval: 2h
+    interval: 1h
```

Comment:
Thanks, this is for testing, so Clair will generate notifications more frequently, I'll open another issue to put interval to 12h in next sprint.